parachains/pallets/collective-content/src/benchmarking.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2023 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! The pallet benchmarks.
+
+use super::{DispatchTimeFor, Pallet as CollectiveContent, *};
+use frame_benchmarking::benchmarks_instance_pallet;
+use frame_support::traits::{EnsureOrigin, UnfilteredDispatchable};
+use sp_core::Get;
+
+fn assert_last_event<T: Config<I>, I: 'static>(generic_event: <T as Config<I>>::RuntimeEvent) {
+	frame_system::Pallet::<T>::assert_last_event(generic_event.into());
+}
+
+benchmarks_instance_pallet! {
+	set_charter {
+		let cid: Cid = b"ipfs_hash".to_vec().try_into().unwrap();
```

Comment:
We may as well bench with an appropriate length. This cid is the polkadot logo:
```suggestion
		let cid: Cid = b"bafkreif2mywzuu2b2uwehi6c6fojgd7dhcqngus5gzy23uangvpa2kc5si".to_vec().try_into().unwrap();
```